magellan-library/src/main/java/com/wealthfront/magellan/ScreenContainer.java

```diff
@@ -6,7 +6,7 @@
 import android.widget.FrameLayout;
 
 /**
- * The container to be used to display the screens using the {@link Navigator}. Must have the id
+ * The container to be used to display the screens using the {@link com.wealthfront.magellan.navigation.Navigator}. Must have the id
```

Comment:
Nit: probably don't want the full classpath here

Reply:
Oh wait, I thought it was kotlin docs, idk what the normal thing to do for javadocs is anymore haha

Reply:
The IDE sorta autocompleted with the full path so I just left it.